### ISSUE TITLE
Independent Publisher: Hebrew Text overlaps in Author Bio

### ISSUE THREAD

**Issue by damiannep** [Automattic/themes#330]
## Steps to replicate
Create a page in Hebrew with author bio enabled. Then open the page and scroll down to the Author Bio enabled on single posts. 

## Result

The text doesn't render properly e.g. https://cld.wthms.co/8eR0eL

## Expected

The text renders correctly in other themes such as Dara: https://cld.wthms.co/CYDLV7

* User report: #7924341-hc
* Follow-up ticket: #1555074-zd
* User's site: https://adam-hofshi.blog/2018/11/09/%D7%90%D7%93%D7%9D-%D7%97%D7%95%D7%A4%D7%A9%D7%99-%D7%A4%D7%95%D7%A1%D7%98-%D7%A8%D7%90%D7%A9%D7%95%D7%9F-%D7%91%D7%9C%D7%95%D7%92-%D7%90%D7%93%D7%9D-%D7%97%D7%95%D7%A4%D7%A9%D7%99/

**Comment by nagpai** [comment]
* The site-posted-on  div seems to remain right floated .. may need to change to left since this is a RTL language

* The Avatar may need to be given a top margin of 20px

Added the following custom CSS

```
@media screen and (min-width: 48.9375em) {
.site-posted-on {
   left: auto;
   right: 0;
}

.entry-author .author-avatar {
   margin-top: 20px;
  }
}
```
Trying to submit my first PR for this :)

**Comment by nagpai** [comment]
I have some new information after my first futile attempt at a PR :-P 

**tl;dr -** The issue seems to be with the broken link of the Photon CDN served RTL CSS file 

After realizing that the CSS rules in the theme's source code rtl.css are correct, my eyes fell on the inspector output for errors

<img width="1013" alt="screenshot 2018-11-10 at 8 36 34 pm" src="https://user-images.githubusercontent.com/4162931/48302871-79388580-e528-11e8-8a2e-0aa1172648aa.png">

It seems that the theme CSS is being hosted by Jetpack Photon CDN which doesnt seem to have the rtl.css on it. 

The URL 
https://s2.wp.com/wp-content/themes/pub/independent-publisher-2/rtl.css?m=1541537380

seems to be broken and thats why the RTL styles are not loading. 

This doesnt seem to be a theme issue, rather its a Jetpack Photon issue. I will check where is the best place to report this

**Comment by nagpai** [comment]
The issue is fixed by the System team. The test site as well as the user's site is now loading the Author Bio box correctly. Closing the issue here.
